migration/iD12/oracle/202404261129_IDEMPIERE-6129.sql

```diff
@@ -0,0 +1,4 @@
+-- IDEMPIERE-6129
+SELECT register_migration_script('202404261129_IDEMPIERE-6129.sql') FROM dual;
+
+ALTER TABLE t_selection_infowindow MODIFY value_string VARCHAR2(4000);
```

Comment:
for oracle we need to define explicitly CHAR on the size:

please apply the following patch:
[pr2338PR1.patch.txt](https://github.com/idempiere/idempiere/files/15311022/pr2338PR1.patch.txt)